src/lang/en.json

```diff
@@ -7,6 +7,8 @@
   "splash.board_name": "Board Name",
   "splash.board_name_example": "Sprint 21 Retrospective",
   "splash.template": "Template",
+  "splash.icebreaking": "Ice Breaking",
```

Comment:
I think "Ice Breaker" would be better here.
I have some friends I can ask about the other languages.

Reply:
Indeed! Nice. Thanks for the help.